lib/src/lib.rs

```diff
@@ -27,3 +27,6 @@ pub mod csv;
 
 #[cfg(feature = "docx")]
 pub mod docx;
+
+#[cfg(feature = "rtf")]
+pub mod rtf;
```

Comment:
Sorry. But unfortanly I do not see rtf.rs file in your PR )

Reply:
There is rtf.rs... (https://github.com/igumnoff/shiva/pull/52/files#diff-e92a95e8406fa49bee15625bb78f7aafc6a3f26bb6672fcf9f7812e2da0dfac7)